lib/mpl_toolkits/basemap/test.py

```diff
@@ -206,5 +206,25 @@ def test():
 
 
 if __name__ == '__main__':
+    # When called with the -v / --verbose commandline parameter, it will
+    # give package dependent version information:
+    #   $ python test.py --verbose
+
+    import argparse # standard module in Python >= 2.7 and >= 3.2
     import unittest
+
+    from mpl_toolkits.basemap.diagnostic import package_versions
+    
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-v', '--verbose',action='count')
+    args = parser.parse_args()
```

Comment:
The use of argparse is overkill here and would interfere with `unittest.main()`, which also processes command-line arguments. Instead, just import sys and inspect `argv` for "-v" and "--verbose".